ui/src/services/NodeVolumesUtils.js

```diff
@@ -78,28 +78,28 @@ export const computeVolumeGlobalStatus = (name, status) => {
   const condStatus = condition?.status;
   const condReason = condition?.reason;
 
-  switch(condStatus) {
-  case 'True':
-    return STATUS_AVAILABLE;
-  case 'False':
-    return STATUS_FAILED;
-  case 'Unknown':
-    switch(condReason) {
-    case 'Pending':
-      return STATUS_PENDING;
-    case 'Terminating':
-      return STATUS_TERMINATING;
+  switch (condStatus) {
+    case 'True':
```

Comment:
Why did the indentation changed? If something have reformatted the code automatically, it should be done in a dedicated commit (separate functional changes from cosmetic ones).